docs/quickstarts/cosign-akv-azure.md

```diff
@@ -0,0 +1,441 @@
+
+    Confirm no signatures before signing
+
+    ```bash
+    cosign tree $IMAGE_SIGNED
+    ```
+    
+    Sign the image
+
+    ```bash
+    cosign sign --key azurekms://$AKV_NAME.vault.azure.net/$KEY_NAME/$KEY_VER --tlog-upload=false $IMAGE_SIGNED
+    ```
+
+    > In this article, use flag `--tlog-upload=false` to skip upload the signature to the transparent log (Rekor by default). 
+    > Sign using key in AKV does not necessarily require the role `Key Vault Crypto Officer`, you can login with another identity and assign the role `Key Vault Crypto User` for signing action.
+
+    Confirm the signature is pushed and associated with the image in ACR
+
+    ```bash
+    cosign tree $IMAGE_SIGNED
+    ```
+
+## Validate signatures at K8s admission control
+
+### Access control
+
+User assigned managed identity is used for Ratify to access ACR and AKV. 
+
+1. Configure environment variables
+
+```bash
+export ID_NAME=<your managed id>
+export ID_RG=<your managed id resource group>
+```
+
+1. Create a user assigned managed identity
+
+    ```bash
+    az identity create --name $ID_NAME --resource-group $ID_RG
+    export OBJECT_ID=$(az identity show --name $ID_NAME --resource-group $ID_RG --query 'principalId' -o tsv)
+    ```
+
+1. Assign `acrpull` role to this identity for accessing ACR
+
+    ```bash
+    az role assignment create --assignee-object-id $OBJECT_ID --role acrpull \
+    --scope "/subscriptions/$SUB_ID/resourceGroups/$ACR_RG/providers/Microsoft.ContainerRegistry/registries/$ACR_NAME"
+    ```
+
+1. Assign `Key Vault Crypto User` role to this identity for accessing AKV
+
+    ```bash
+    az role assignment create --role "Key Vault Crypto User" --assignee $OBJECT_ID \
+    --scope "/subscriptions/$SUB_ID/resourceGroups/$AKV_RG/providers/Microsoft.KeyVault/vaults/$AKV_NAME"
+    ```
+
+### Set up your AKS cluster
+
+1. Configure environment variables
+
+    ```bash
+    export CLIENT_ID=$(az identity show --name  $ID_NAME --resource-group $ID_RG --query 'clientId' -o tsv)
+    export TENANT_ID=$(az identity show --name $ID_NAME --resource-group $ID_RG --query tenantId -o tsv)
+    ```
+
+1. Update the AKS cluster with OIDC and workload identity support
+
+    ```bash
+    az aks update -g $AKS_RG -n $AKS_NAME --enable-oidc-issuer --enable-workload-identity
+    ```
+
+1. Create federate credential for your managed identity
+
+    ```bash
+    export AKS_OIDC_ISSUER="$(az aks show -n $AKS_NAME -g $AKS_RG --query "oidcIssuerProfile.issuerUrl" -o tsv)"
+
+    az identity federated-credential create --name ratify-federated-credential --identity-name $ID_NAME -g $ID_RG --issuer $AKS_OIDC_ISSUER --subject system:serviceaccount:"gatekeeper-system":"ratify-admin"
+    ```
+
+### Install Ratify
+
+1. AKS with Azure policy enabled
+   
+   Make sure AKS image integrity policy is not assigned to your AKS cluster.
+
+   ```bash
+   helm install ratify oci://ghcr.io/deislabs/ratify-chart-dev/ratify --atomic \
```

Comment:
I don't think we should point to the dev chart here? We should point chart repo `ratify/ratify`